zerver/migrations/0310_jsonfield.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.1.5 on 2021-01-10 11:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    """
+    This doesn't actually run any SQL, it's for Django's internal
+    tracking of changes to models only.
+    django.contrib.postgres.fields.JSONField is deprecated as of Django 3.1
+    and should be replaced by models.JSONField which offers the same functionality.
+    """
+
+    dependencies = [
+        ('zerver', '0309_userprofile_can_create_users'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='userprofile',
+            name='zoom_token',
+            field=models.JSONField(default=None, null=True),
+        ),
+    ]
```

Comment:
For this, could we just change the original definition of `zoom_token` instead?  If the SQL is really the same, that should be correct.  (I'm thinking ahead to when Django deletes the old `django.contrib.postgres.fields.JSONField`, which will probably require us to make that change to the existing migration anyway)

Reply:
I did it as a new migrations, because that's what they're actually recommending in https://docs.djangoproject.com/en/3.1/releases/3.1/: 
> If your project uses django.contrib.postgres.fields.JSONField, plus the related form field and transforms, you should adjust to use the new fields, and generate and apply a database migration

So I assume they intend to keep this working in migrations.

Reply:
Yeah, I think this will end up being slightly annoying when we do a `squashmigrations` later, but only very slightly.